pkgs/development/python-modules/mdformat-wikilink/default.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  mdformat,
+  poetry-core,
+  pytest-cov,
+  pytestCheckHook,
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "mdformat-wikilink";
+  version = "0.2.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "tmr232";
+    repo = "mdformat-wikilink";
+    rev = "v${version}";
+    hash = "sha256-KOPh9iZfb3GCvslQeYBgqNaOyqtWi2llkaiWE7nmcJo=";
+  };
+
+  nativeBuildInputs = [
+    poetry-core
+    pytest-cov
+  ];
+
+  propagatedBuildInputs = [ mdformat ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  pythonImportsCheck = [ "mdformat_wikilink" ];
+
+  meta = with lib; {
+    description = "An mdformat plugin for ensuring that wiki-style links are preserved during formatting.";
```

```suggestion
    description = "mdformat plugin for ensuring that wiki-style links are preserved during formatting";
```
See https://nixos.org/manual/nixpkgs/unstable/#var-meta-description